applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -1639,6 +1639,276 @@ Complete the following steps to test the functionality provided by the :ref:`SLM
       #XPING: average 0.650
       OK
 
+Testing GPS AT commands
+-----------------------
+
+Before you test the GPS AT commands, check the following configuration values:
+
+* :option:`CONFIG_SUPL_CLIENT_LIB` - must be enabled to use the :ref:`supl_client`
+* :option:`CONFIG_SLM_SUPL_SERVER` - use the default ("supl.google.com") or specify a different server
+* :option:`CONFIG_SLM_SUPL_PORT` - use the default (7276) or specify a different port
+
+Complete the following steps to test the functionality provided by the :ref:`SLM_AT_GPS`:
+
+1. Test standalone GPS:
+
+   a. Enable support for GNSS in the modem.
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT+CFUN=0**
+         OK
+
+         **AT%XSYSTEMMODE=0,0,1,0**
+         OK
+
+   .. modem_config_start
+
+   b. Configure the modem.
+      The required commands depend on the version of the nRF9160 DK.
+
+      For nRF9160 DK v0.9.0:
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT%XMAGPIO=1,0,0,1,1,1574,1577**
+         OK
+
+      For nRF9160 DK up to v0.10.x:
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT%XCOEX0=1,1,1570,1580**
+         OK
+
+   .. modem_config_end
+
+   c. Turn on the modem.
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT+CFUN=1**
+         OK
+
+   #. Start GPS and observe the output.
+      Then check the GPS status.
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT#XGPS=1,3**
```

Comment:
What does the "3" mean?

Reply:
3 is the value for the "mask" field. 

When the ``<status>`` value is 1, the ``<mask>`` value syntax appears as follows:

* Bit 0 - Global Positioning System Fix Data
* Bit 1 - Geographic position latitude/longitude and time
* Bit 2 - DOP and active satellites
* Bit 3 - Satellites in view
* Bit 4 - Recommended minimum specific GPS/transit data

3 means bit 0 and bit 1 are used.